### PR TITLE
Milestone: Add crud forms to blog App

### PR DESCRIPTION
**Pull request by Felix45**
- Added web forms and crud method for creating a post
- Added web forms and crud method for creating a comment
- Added web forms and crud method for adding a like to a post 
- Add navigation links to all the forms created

### CODE REVIEW

File: app/controllers/posts_controller.rb [hunk 1]
```diff
@@ -3,7 +3,38 @@ def index
...
+  end
+
   def show
     @post = Post.find(params[:id])
   end
+
+  def post_params
+    params.require(:post).permit(:title, :text)
+  end
 end
```
Comment (OLIPLICHE) [inline app/controllers/posts_controller.rb:40]: ## Nice for using Strong parameters
- I suggest you mention `private` in these private methods. keep applying this in all controllers
- See an Example from this link 👉 [Ruby on Rails Strong parameters](https://guides.rubyonrails.org/action_controller_overview.html#strong-parameters)

File: config/routes.rb [hunk 2]
```diff
@@ -3,6 +3,9 @@
   
   root 'users#index'
   resources :users, only: [:index, :show] do
-    resources :posts, only: [:index, :show]
+    resources :posts, only: [:index, :new, :create, :show] do
```
Comment (OLIPLICHE) [inline config/routes.rb:6]: - Nice
- 
Capture

File: app/views/users/show.html.erb [hunk 3]
```diff
@@ -2,6 +2,9 @@
 <div class="row">
   <div class="col-md-2 col-sm-12">
```
Comment (OLIPLICHE) [inline app/views/users/show.html.erb:3]: ## Congratulations 
Capture

**Review by OLIPLICHE — APPROVED** [review]
Hi @Felix45 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Linter is green (the last commit have a green check mark (✅)) 👏
- README file is descriptive 👏
- Commit messages are meaningful 👏 👏
- Gitflow is correctly used (Pull Request is made from the feature branch into the dev branch) 👏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/users/show.html.erb (hunk 3 above)
Comment (Felix45) [reply app/views/users/show.html.erb:3]: @OLIPLICHE Thanks

File: app/controllers/posts_controller.rb (hunk 1 above)
Comment (Felix45) [reply app/controllers/posts_controller.rb:40]: @OLIPLICHE thanks for mentioning this, I will implement this suggestion

**Pull request merged**